docs/reference/pages/editing_api.rst

```diff
@@ -18,232 +18,248 @@ Defining Panels
+            content_panels = Page.content_panels + [
+                PageChooserPanel('related_page', 'demo.PublisherPage'),
+            ]
 
-Django's default behavior is to "cascade" deletions through a ForeignKey relationship, which is probably not what you want happening. This is why the ``null``, ``blank``, and ``on_delete`` parameters should be set to allow for an empty field. (See `Django model field reference (on_delete)`_ ). ``ImageChooserPanel`` takes only one argument: the name of the field.
+    ``PageChooserPanel`` takes two arguments: a field name and an optional page type. Specifying a page type (in the form of an ``"appname.modelname"`` string) will filter the chooser to display only pages of that type.
 
-.. _Django model field reference (on_delete): https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.ForeignKey.on_delete
+ImageChooserPanel
+-----------------
 
-Displaying ``Image`` objects in a template requires the use of a template tag. See :ref:`image_tag`.
+.. class:: ImageChooserPanel(field_name)
 
+    One of the features of Wagtail is a unified image library, which you can access in your models through the ``Image`` model and the ``ImageChooserPanel`` chooser. Here's how:
 
-Documents
----------
+    .. code-block:: python
 
-For files in other formats, Wagtail provides a generic file store through the ``Document`` model:
+      from wagtail.wagtailimages.models import Image
+      from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
 
-.. code-block:: python
 
-  from wagtail.wagtaildocs.models import Document
-  from wagtail.wagtaildocs.edit_handlers import DocumentChooserPanel
-  # ...
-  class BookPage(Page):
-    book_file = models.ForeignKey(
-      'wagtaildocs.Document',
-      null=True,
-      blank=True,
-      on_delete=models.SET_NULL,
-      related_name='+'
-    )
-
-  BookPage.content_panels = [
-    DocumentChooserPanel('book_file'),
-    # ...
-  ]
+      class BookPage(Page):
+          cover = models.ForeignKey(
+              'wagtailimages.Image',
+              null=True,
+              blank=True,
+              on_delete=models.SET_NULL,
+              related_name='+'
+          )
 
-As with images, Wagtail documents should also have the appropriate extra parameters to prevent cascade deletions across a ForeignKey relationship. ``DocumentChooserPanel`` takes only one argument: the name of the field.
+          content_panels = Page.content_panels + [
+              ImageChooserPanel('cover'),
+          ]
 
-Documents can be used directly in templates without tags or filters. Its properties are:
+    Django's default behavior is to "cascade" deletions through a ForeignKey relationship, which is probably not what you want happening. This is why the ``null``, ``blank``, and ``on_delete`` parameters should be set to allow for an empty field. (See `Django model field reference (on_delete)`_ ). ``ImageChooserPanel`` takes only one argument: the name of the field.
```

Comment:
"which is probably not what you want happening" -> "which may not be what you want"